DaemonRenderer/Source/Include/Windowing/Screen.hpp

```diff
@@ -26,47 +26,72 @@
 
 #include <vector>
 
-#include "Utility.hpp"
+#define GLFW_INCLUDE_NONE
+
+#include <glfw/glfw3.h>
+
+#include "Vector/Vector.hpp"
+
+#include "Windowing/WindowParams.hpp"
 
 BEGIN_DAEMON_NAMESPACE
 
+struct VideoMode
+{
+    DAEint32 width;
+    DAEint32 height;
+    DAEint32 red_bits;
+    DAEint32 green_bits;
+    DAEint32 blue_bits;
+    DAEint32 refresh_rate;
+};
+
+struct GammaRamp
```

Comment:
This struct should go in its own file

Reply:
Will be done during the WindowManager cleanup.